### ISSUE TITLE
Page field custom PHP code with missing ";" error

### ISSUE THREAD

**Issue by somatonic** [ryancramerdesign/ProcessWire#1177]
```
return $page
```

When entering a code without ; it shows error on top and red error at html bottom.

**Comment by ryancramerdesign** [comment]
That's a PHP parse error. PHP lines are required to end with a semi colon. PHP does the parsing on this one, not PW, so it has to follow PHP rules.

**Comment by somatonic** [comment]
Please go and enter such a code and hit save. Thx.

**Comment by somatonic** [comment]
When you enter a code and choose one of the PageListSelect or Autocomplete and save it shows 

"Parse error: syntax error, unexpected $end in /home/www-update/verband.update.ch/wire/modules/Inputfield/InputfieldPage/InputfieldPage.module(209) : eval()'d code on line 1"

As plain Text above the html on top (breaks dropdown usage) and the same message highlighted red error shows at the bottom.

**Comment by ryancramerdesign** [comment]
Okay, I just tried what you suggested: putting invalid PHP code into a field asking for valid PHP code that's going to be eval'd. I got exactly what I expected, a PHP error. :) It's a fatal PHP error so it's showing up when it occurs, which is why you see it at the top. PW also reiterates any PHP errors in red at shutdown, in case the error got trapped in a non-visible block in the markup. So the duplication is very much intentional, and you'll see it do the same thing on the front-end of your site when debug mode is on.
